### PR TITLE
Should date type `parseValue` take a string and return a Date?

### PR DESCRIPTION
**Pull request by jkempff**
I have a mongoose model, that contains a field of type `Date`. Manual tests of `add${ModelName}Mutation` in graphiql worked fine.

But then, having it all wrapped up in Relay, `coerceDate` throws the "Field error: value is not an instance of Date" Error. (At this point, `value` is the dates ISO String, so `instanceof Date` always fails)

With my fork, it works fine. I'm not entirely sure, if this is the root of my problem, but it makes sense to me.

Basically, `parseValue` now takes a String and tries to turn it into a valid Date object.

### CODE REVIEW

**Comment by jkempff** [comment]
I actually just copied the changes from this pull-request https://github.com/soundtrackyourbrand/graphql-custom-datetype/pull/2 that hasn't been merged yet. (But the project seems to sleep)

**Comment by tothandras** [comment]
@jkempff Thank you! I've [merged it](https://github.com/RisingStack/graffiti-mongoose/commit/4a0d254f8b06510240d823791830c906275a9bd5), but changed the commit message and fixed a test. The test case you have included was already there.

**Comment by tothandras** [comment]
I will release your fix tomorrow.

**Comment by jkempff** [comment]
Awesome! Thank you for the quick reply.

**Comment by tothandras** [comment]
@jkempff I've released it on npm under version `5.0.2`. Thanks for your contribution! ;)
